### DISCUSSION TITLE
centos7安装Adgurad home的仪表盘界面的疑问

### DISCUSSION THREAD

**Discussion by 1014951275** [AdguardTeam/AdGuardHome#3629] [Q&A]
使用esxi重新安装centos 7，在centos 7里面安装Adgurad home过滤器，过滤使用正常，虚拟机重启后仪表盘界面里面的数据被清零，查询日志有数据，这是那里出问题了
![image](https://user-images.githubusercontent.com/90028761/133993895-8e2b3fab-ac5c-41e1-b5e4-6249be0ec390.png)

**Comment by ameshkov** [comment]
你用什么虚拟机？是不是 docker？

**Reply by 1014951275** [reply]
vmware esxi 6.7虚拟机安装centos 7 linux系统，并没有在docker里面安装
安装代码
![微信截图_20210921031432](https://user-images.githubusercontent.com/90028761/134061107-b74e84e9-d72c-4773-853c-9574b012b92e.png)
端口已放行，过滤规则正常，就是这个仪表盘界面里面的数据重启后就会清零，之前还没有重新安装的时候是正常的，想弄清原因

**Reply by ameshkov** [reply]
你怎么重启？是`./AdGuardHome -s restart`吗？

试试启用 AGH verbose 水平的日志：https://github.com/AdguardTeam/AdGuardHome/wiki/FAQ#verboselog

**Reply by 1014951275** [reply]
直接虚拟机reboot重启，或者有时候esxi整机重启

**Reply by ameshkov** [reply]
AGH正常的关掉的话 （`./AdGuardHome -s stop`）它会保存数据，不然有的数据流在内存。

**Reply by 1014951275** [reply]
这就奇怪，上一台虚拟机没有这问题，设置7天保存数据正常，不管什么断电重启都有数据

**Reply by ameshkov** [reply]
哦，很奇怪。坦白来说，我不明白这个问题是什么。有可能和那个虚拟机有关系。
